src/scripts/rhsm/get_access_token.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import sys
+import json
+
+args = json.loads(sys.argv[1], strict=False)
+offline_token = args["offlineToken"]
+
+data = {
+    "grant_type": "refresh_token",
+    "client_id": "rhsm-api",
+    "refresh_token": offline_token
+}
+
+# Handle HTTP request failure
+try:
+    r = requests.post(url="https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token", data=data)
+    r.raise_for_status()
+except requests.exceptions.RequestException as error:
+    sys.exit(error)
+
+# Handle RHSM API failure
+ret_obj = r.json()
+if "error" in ret_obj:
+    sys.exit(ret_obj["error_description"])
+
+print(ret_obj["access_token"])
```

Comment:
Hmm. I think it would end up being uncessarily more complex. Firstly, if there is a runtime error in python, that is eruttuner as aa objet { problem: null, exit_status: 1, exit_signal: null, message: "some message" }. Os if I return a JSOn, then message would contain another JSON with some object in it, instead of a message.
Also request exceptions `requests.exceptions.RequestException`, which I'm also trying to handle, are not serializable, so to make it work that's additional complexity.

Reply:
OK, makes sense. Then please just fix the `error` vs. `error_description` mismatch, as you test for one and then assume the other.